packages/react-app/src/App.jsx

```diff
@@ -23,15 +23,15 @@ const DEBUG = true;
 const NETWORKCHECK = true;
 
 // Add more networks as the dapp expands to more networks
-const configuredNetworks = ["mainnet", "rinkeby", "xdai", "matic", "mainnetAvalanche"];
+const configuredNetworks = ["rinkeby"];
 if (location.hostname === "localhost" || location.hostname === "127.0.0.1") {
   configuredNetworks.push("localhost");
 }
 
 const cachedNetwork = window.localStorage.getItem("network");
 if (DEBUG) console.log("📡 Connecting to New Cached Network: ", cachedNetwork);
 /// 📡 What chain are your contracts deployed to?
-let targetNetwork = NETWORKS[cachedNetwork || "mainnet"]; // <------- select your target frontend network (localhost, rinkeby, xdai, mainnet)
+let targetNetwork = NETWORKS[cachedNetwork || "rinkeby"]; // <------- select your target frontend network (localhost, rinkeby, xdai, mainnet)
```

Comment:
No need to set this. You can switch your local network switch to rinkeby

Reply:
Was giving an error since the contract isn't deployed on mainnet, we can change this back once pushing to production